web/.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   plugins: ['@typescript-eslint', 'react-hooks'],
   extends: ['plugin:react/recommended', 'plugin:@typescript-eslint/recommended'],
   rules: {
-    'max-len': ['warn', { code: 120 }],
+    'max-len': ['warn', { code: 130 }],
```

Comment:
I'd say the same as for the gocyclo linter.

Reply:
another question to ask is : what is a good value here? I've raised to 130, to be honest I first wanted to raise it to 150. We need to find something that is reasonable to all of us. Personally, given my screen resolution, I'm comfortable up to 150.

Reply:
I'm usually more confortable with short values (e.g. 100) because it minimizes the distance that my eyes need to travel and I can also split windows vertically with a reasonable font size.

But that's up to the rest of the team, I can get adapted to any configuration.

Reply:
I don't have any strong opinion here but I also have a preference for short lines

Reply:
that's fine, I reverted this change